vars/githubApiCall.groovy

```diff
@@ -46,12 +49,18 @@ def call(Map params = [:]){
       def key = "${token}#${url}"
       if(cache["${key}"] == null || noCache){
         log(level: 'DEBUG', text: "githubApiCall: get the JSON from GitHub.")
+        def parameters = [url: url, headers: headers]
         if(data) {
           log(level: 'DEBUG', text: "gitHubApiCall: found data param. Switching to ${method}")
           headers.put("Content-Type", "application/json")
-          json = httpRequest(url: url, method: method, headers: headers, data: toJSON(data).toString())
+          // toJSON(data).toString() caused some issues with metadata entries in the output.
+          json = httpRequest(parameters + [method: method, data: new groovy.json.JsonBuilder(data).toPrettyString()])
```

Comment:
For some reason the JSON is transformed with some metadata, such as bytes, name, string... So as it's changed then it might be required to validate the consumers of the `githubApiCall` are not broken. 🔥